### PR TITLE
Deaf ghost/guest.

### PR DESCRIPTION
**Pull request by bbfsdev**

### CODE REVIEW

File: src/apps/MobileApp/MobileClient.js [hunk 1]
```diff
@@ -1051,6 +1051,10 @@ class MobileClient extends Component {
     // Subscribes selectively to different stream types |subscribeToVideo|, |subscribeToAudio|, |subscribeToData|.
     // This is required to stop and then start only the videos to save bandwidth.
     makeSubscription = (newFeeds, feedsJustJoined, subscribeToVideo, subscribeToAudio, subscribeToData) => {
+      const {user: {role}} = this.state;
```
Comment (edoshor) [inline src/apps/MobileApp/MobileClient.js:1054]: move to call site. Should not be part of `makeSubscription`. There is an argument for that: `subscribeToAudio`

**Review by edoshor — CHANGES_REQUESTED** [review]

File: src/apps/MobileApp/MobileClient.js (hunk 1 above)
Comment (bbfsdev) [reply src/apps/MobileApp/MobileClient.js]: Fixed.

**Review by edoshor — APPROVED** [review]

**Comment by amnonbb** [comment]
As I understand ghost/guest does not subscribe on audio. Right?

**Comment by bbfsdev** [comment]
Yes, ghost and guest don't subscribe to audio.

**Pull request merged**